modules/updating-eus-to-eus-upgrade-cli.adoc

```diff
@@ -0,0 +1,145 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing-eus-eus-upgrade.adoc
+
+:_content-type: PROCEDURE
+[id="updating-eus-to-eus-upgrade-cli_{context}"]
+= EUS-to-EUS update using the CLI
+
+.Prerequisites
+
+* Verify that machine config pools are unpaused.
+* Update OpenShift CLI `oc` to the target version before each update.
```

Comment:
We should still switch these to the [term-glossary-approved version](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#openshift-cli) "the OpenShift CLI (`oc`)". Here and in the `IMPORTANT` box below.